kolibri/plugins/openid_connect/openid_connect_settings.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+from kolibri.deployment.default.settings.base import AUTHENTICATION_BACKENDS
+from kolibri.deployment.default.settings.base import INSTALLED_APPS
+
+INSTALLED_APPS += ["mozilla_django_oidc"]
+
+# OIDC SETUP
+AUTHENTICATION_BACKENDS += [
+    "kolibri.plugins.openid_connect.auth.OIDCKolibriAuthenticationBackend"
+]
+OIDC_URL = os.environ.get("API_URL", "http://127.0.0.1:5002/oauth")
```

Comment:
Would it be good to set this via the options.ini? (not currently possible for plugins to add to it, but wondering if it might be worthwhile).

Reply:
yes, it would be much better, but I didn't want to add more code in the Kolibri 'core'. Is it possible to add entries to the options.ini file from a plugin?

Reply:
Not yet, but I am prototyping something now to allow this. Just wanted to make sure that it would be of use!

Reply:
When we return to this plugin, this is now possible!